db.json

```diff
@@ -6495,20 +7631,21 @@
   "application/xliff+xml": {
     "source": "iana",
     "compressible": true,
-    "extensions": ["xlf"]
+    "extensions": ["xlf","xliff"]
   },
   "application/xml": {
     "source": "iana",
     "compressible": true,
-    "extensions": ["xml","xsl","xsd","rng"]
+    "extensions": ["xml","xsl","xbl","xsd","rng"]
   },
```

Comment:
https://www.iana.org/assignments/media-types/application/xml only lists .xml 

.xbl is part of XUL , a UI language which failed standardisation, and doesnt have a lot of usage these days (sob).